src/libraries/shims/ApiCompatBaseline.netcoreapp.netstandard.txt

```diff
@@ -387,6 +387,79 @@ CannotMakeMemberNonVirtual : Member 'System.Data.Constraint.ConstraintName.set(S
 CannotMakeMemberNonVirtual : Member 'System.Data.Constraint.Table.get()' is non-virtual in the implementation but is virtual in the contract.
 CannotMakeMemberNonVirtual : Member 'System.Data.Constraint.ToString()' is non-virtual in the implementation but is virtual in the contract.
 CannotMakeMemberNonVirtual : Member 'System.Data.Constraint._DataSet.get()' is non-virtual in the implementation but is virtual in the contract.
+Compat issues with assembly System.Data.SqlClient:
```

Comment:
I am not sure whether this is the best way to suppress these.

Reply:
What contract is complaining about these?  The System.Data.dll shim?  What implications does that have for libraries built against the netfx surface area trying to run on core?

Reply:
We need those shims to continue to work.  You can restore an previous version of the package to produce the correct shims.

Reply:
> You can restore an previous version of the package to produce the correct shims.

I have added it to https://github.com/dotnet/runtime/pull/2275/files#diff-d99c84c0369652bea55f739641b3ef74R40 . Is this the right place to restore the package?